settings

```diff
@@ -0,0 +1 @@
+{"theme": "Default"}
```

Comment:
Please use the proper API for settings. (gui_helpers.settings), [documentation](https://doc.qt.io/qtforpython/PySide6/QtCore/QSettings.html) is available here. It is also used by the DownloadDialog and SendDialog if you'd like to see how to use it.

Reply:
Once this is resolved, the function from utils.py can probably be removed too.

Reply:
yup. To be frank, I didn't have any experience working with this but it's much more elegant than what I proposed. Should be fixed in the newest commit (also yes, the utils function was not needed after all)